packages/components/src/components/numpad/numpad.tsx

```diff
@@ -49,15 +50,15 @@ const Numpad = ({
     onValueChange,
     onValidate,
 }: TNumpad) => {
-    const formatNumber = (v: string) => (typeof format === 'function' ? format(v) : Number(v));
-    const isFloat = (v: string) => new RegExp(/\./).test(String(v));
+    const formatNumber = (v: string | number) => (typeof format === 'function' ? format(v.toString()) : v);
```

Comment:
Can u please re-check this? I belive in all places u r just passing values to `formatNuber` function. Current implementation has no scenario of passing a function. So do u need this function?

Reply:
@likhith-deriv this is out of the scope of this card, changing this will break things, it's better to fix it later in a separate card :pray:

Reply:
The purpose here was just ts migration! by changing this we may need to refactor other parts too